.github/workflows/ci.yml

```diff
@@ -39,9 +40,11 @@ jobs:
           pip install -r api/requirements.txt
 
       - name: Run tests
+        env:
+          DATABASE_HOST: localhost
```

Comment:
Nice! I like this a lot better than using a separate github specific settings file that is only defining the DATABASE_HOST.